README.md

```diff
@@ -15,12 +15,31 @@ This project hosts the Java client library for the Google Ads API.
 
 ## Maven artifacts
 
+The default maven artifact is `google-ads` at the latest version. The following dependency may be
+added to include the Google Ads Java library.
```

Comment:
Please word this as instructions, e.g., "Add the following dependency to your project to include the Google Ads API library for Java."